README.md

```diff
@@ -1,6 +1,21 @@
+<div align="center">
+
+<img src="assets/codeyProduction.png" width="250" height="250" align="left">
```

Comment:
Should the assets be in the git ignore?

Reply:
I don't think so? I think if people want to add or delete things from the assets folder, then those changes should be reflected for everyone.